WordPress/Classes/ViewRelated/Stats/JetpackLoginViewController.swift

```diff
@@ -62,22 +64,34 @@ class JetpackLoginViewController: UIViewController {
         setupControls()
     }
 
+    override func willTransition(to newCollection: UITraitCollection, with coordinator: UIViewControllerTransitionCoordinator) {
+        super.willTransition(to: newCollection, with: coordinator)
+        hideImageView(with: newCollection)
+    }
+
     // MARK: - Configuration
 
     /// One time setup of the form textfields and buttons
     ///
     fileprivate func setupControls() {
-        switch promptType {
-        case .stats:
-            jetpackImage.image = UIImage(named: "wp-illustration-stats")
-        case .notifications:
-            jetpackImage.image = UIImage(named: "wp-illustration-notifications")
-        }
+        jetpackImage.image = promptType.image
+        hideImageView(with: traitCollection)
+
         descriptionLabel.font = WPStyleGuide.fontForTextStyle(.body)
         descriptionLabel.textColor = WPStyleGuide.darkGrey()
+
+        tacButton.titleLabel?.numberOfLines = 0
+
+        faqButton.titleLabel?.font = WPStyleGuide.fontForTextStyle(.subheadline, fontWeight: .medium)
+        faqButton.setTitleColor(WPStyleGuide.wordPressBlue(), for: .normal)
+
         updateMessageAndButton()
     }
 
+    private func hideImageView(with collection: UITraitCollection) {
```

Comment:
This is probably just me being pedantic but this method will hide or show depending on the trait collection, yeah?  Maybe we could rename the method to more clearly communicate that behavior?